common/components/PrintableWallet/index.tsx

```diff
@@ -1,71 +1,53 @@
 import { PaperWallet } from 'components';
-import PrivKeyWallet from 'libs/wallet/privkey';
-import React, { Component } from 'react';
+import { IFullWallet } from 'ethereumjs-wallet';
+import React from 'react';
```

Comment:
Generally I like to have some sort of order of imports. React first, then Redux and HOCs, then utils, and components at the end. It doesn't look like we have a specified order for imports, but something we could keep in mind for the future.